src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -131,11 +131,15 @@ private string TryGetText(SyntaxToken token, SemanticModel semanticModel, Docume
             return string.Empty;
         }
 
-        private bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
+        private static bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
         {
             if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
                 token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
-                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken) ||
+                token.IsKind(SyntaxKind.InterpolatedSingleLineRawStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedSingleLineRawStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedMultiLineRawStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedMultiLineRawStringEndToken))
```

Comment:
Consider refactoring this long list of syntax kinds into an extension in https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/CSharpExtensions.cs